charts/argo-events/values.yaml

```
@@ -1,163 +1,403 @@
+    successThreshold: 1
+    # -- Number of seconds after which the [probe] times out
+    timeoutSeconds: 1
+  livenessProbe:
+    # -- Minimum consecutive failures for the [probe] to be considered failed after having succeeded
+    failureThreshold: 3
+    # -- Number of seconds after the container has started before [probe] is initiated
+    initialDelaySeconds: 10
+    # -- How often (in seconds) to perform the [probe]
+    periodSeconds: 10
+    # -- Minimum consecutive successes for the [probe] to be considered successful after having failed
+    successThreshold: 1
+    # -- Number of seconds after which the [probe] times out
+    timeoutSeconds: 1
+
+  # -- Additional volumes to the events controller pod
+  volumes: []
+
+  # -- Additional volumeMounts to the events controller main container
+  volumeMounts: []
+
   # -- [Node selector]
   nodeSelector: {}
-  # -- Labels to be added to event source controller pods
-  podLabels: {}
-  # -- Priority class for the event source controller
-  priorityClassName: ""
   # -- [Tolerations] for use with node taints
   tolerations: []
-  # -- Assign custom [affinity] rules to the event source controller
+  # -- Asign custom [affinity] rules to the deployment
   affinity: {}
-  # -- Resource limits and requests for the event source controller pods
+
+  # -- Assign custom [TopologySpreadConstraints] rules to the events controller
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+  ## If labelSelector is left out, it will default to the labelSelector configuration of the deployment
+  topologySpreadConstraints: []
+  # - maxSkew: 1
+  #   topologyKey: topology.kubernetes.io/zone
+  #   whenUnsatisfiable: DoNotSchedule
+
+  # -- Priority class for the events controller pods
+  priorityClassName: ""
+
+  # -- Resource limits and requests for the events controller pods
   resources: {}
-  # -- Event source controller container-level security context
-  containerSecurityContext: {}
-
-eventbusController:
-  # -- Event bus controller name
-  name: eventbus-controller
-  # -- Repository to use for the event bus controller
-  image: argoproj/argo-events
-  # -- Overrides the image tag
-  # @default -- `""` (default is the chart appVersion)
-  tag: ""
-  # -- The number of event bus controller pods to run
-  replicaCount: 1
-  # -- Additional environment variables to pass to event bus controller
-  extraEnv: []
-    # - name: DEBUG_LOG
-    #   value: "true"
-
-  # -- Annotations to be added to event bus controller pods
+  #  limits:
+  #    cpu: 500m
+  #    memory: 512Mi
+  #  requests:
+  #    cpu: 250m
+  #    memory: 256Mi
+
+  # -- Additional containers to be added to the events controller pods
+  extraContainers: []
+
+  # -- Init containers to add to the events controller pods
+  initContainers: []
+
+  serviceAccount:
+    # -- Create a service account for the events controller
+    create: true
+    # -- Service account name
+    name: argo-events
```

```suggestion
    name: ""
```

What about using the template and set this to empty string?